app/utils/DateOfChangeHelper.scala

```diff
@@ -16,15 +16,42 @@
 
 package utils
 
+import controllers.hvd.routes
 import models.status.{ReadyForRenewal, RenewalSubmitted, SubmissionDecisionApproved, SubmissionStatus}
 import models.tradingpremises.TradingPremises
 import org.joda.time.LocalDate
+import play.api.Logger
 import play.api.Play.current
 import play.api.i18n.Messages
 import play.api.i18n.Messages.Implicits._
+import play.api.mvc.Call
 
 trait DateOfChangeHelper {
 
+  case class DateOfChangeRedirect(call: Call)
+
+  object DateOfChangeRedirect {
+
+    val CHECK_YOUR_ANSWERS:String = "1"
```

Comment:
Is there any chance we can change this to lowerCamelCase just to keep consistency throughout the service?